src/lib/Card/Card.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import { Meta, StoryObj } from "@storybook/react";
+import React from "react";
+import { Card } from "./Card";
+import {
+  CardImageBorderRadius,
+  CardImageSize,
+  CardProps,
+  CardVariant,
+} from "./Card.types";
+
+const meta: Meta<typeof Card> = {
+  component: Card,
+  title: "Card",
+  tags: ["autodocs"],
+};
+export default meta;
+
+type Story = StoryObj<typeof Card>;
+
+export const SampleComponent: Story = (args: CardProps) => {
+  return <Card {...args} />;
+};
+
+SampleComponent.parameters = {
+  docs: {
+    canvas: { sourceState: "shown" },
+    story: { height: "500px" },
+  },
+};
+
+SampleComponent.args = {
+  children: <div style={{ width: "300px" }}>Team lead</div>,
+  title: "MangoBurger",
+  imageSize: CardImageSize.LARGE,
```

Comment:
Not a required change, but the documentation in the typings for this component mentions that the default state of `imageSize` is medium, however here we have it set to large. For consistency it might be good to update this to Medium or the documentation for this prop to specify Large as the default

Reply:
Yeah, good point. I set defaults for the component based on the existing reports and contest tiles, but then I wanted to view it as the team member card style, so I set the storybook sample params to match that. I'll change it.